doc/modules/bayes_expiry.md

```diff
@@ -46,47 +44,43 @@ Every minute `bayes expiry` module executes an expiry step. On each step it chec
 ## Expiration modes
 
 ### Default expiration mode (before 2.0)
-`Default` mode has been removed in Rspamd 2.0 as it has no advantages over `lazy` mode.
+The `default` mode has been removed in Rspamd 2.0 as it offers no benefits compared to the`lazy` mode.
 
 Operation:
 - Extend a `significant` token's lifetime: update token's TTL every time to `expire` value.
 - Do nothing with an `insignificant` or `infrequent` token.
 - Discriminate a `common` token: reset TTL to a low value (10d) if the token has greater TTL.
 
 Disadvantages:
-- Statistics cannot kept off-line longer than `expire` time. TTLs need to be periodically updating by `bayes expiry` module. This means it requires special procedures to backup statistics. If you just make a copy of the `*.rdb` file, you should know that it has a "shelf-life". If you restore it after `expire` time, all tokens will be expired.
-- Constant TTL updating of `significant` tokens is unnecessary if no eviction policy is configured in Redis that assumes `significant` tokens eviction.
+- Statistics must not be stored offline for longer than the`expire` time. The Bayes Expiry module must periodically update their TTLs, which means special backup procedures are required. Simply copying the `*.rdb` file will result in its expiration after the `expire` time has passed.
+- If no eviction policy is set in Redis that targets `significant` tokens, constant updating of their TTLs is not necessary.
 
 ### Lazy expiration mode (1.7.4+)
-`Lazy` mode is the only expiration mode since Rspamd 2.0.
+The `lazy` mode is the only expiration mode since Rspamd 2.0.
 
-Operation:
-- Make a `significant` token persistent if it has TTL.
-- Set TTL of an `insignificant` or `infrequent` token to `expire` value if its current TTL is greater than `expire`.
-- Discriminate a `common` token: resets TTL to a low value (10d) if the token has greater TTL.
+This mode ensures that `significant` tokens with a TTL are persistently kept, while TTL of `insignificant` or `infrequent` tokens is reduced to the `expire` value if its current TTL exceeds `expire`. `Common` tokens are discriminated by resetting their TTL to a lower value of 10 days, should their TTL exceed this threshold.
```

Comment:
It is hard to understand what "are persistently kept" means.  The module sets significant tokens TTLs to -1, i.e. makes them persistent if they are not.

", should their TTL exceed this threshold." I would use "if" here as it easier to understand for non-natives.